ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift

```diff
@@ -613,8 +625,12 @@ extension SceneDelegate {
       urlToOpen = nil
     }
 
-    scene.userActivity = BrowserState.userActivity(for: windowId.uuidString, isPrivate: false)
-    BrowserState.setWindowInfo(for: scene.session, windowId: windowId.uuidString, isPrivate: false)
+    scene.userActivity = BrowserState.userActivity(for: windowId.uuidString, isPrivate: isPrivate)
+    BrowserState.setWindowInfo(
+      for: scene.session,
+      windowId: windowId.uuidString,
+      isPrivate: isPrivate
```

Comment:
Note: These lines have nothing to do with it and don't cause issues. They were updated only because I noticed them.

Reply:
Why are these changed if they have nothing to do with the other changes? You also now pass in `isPrivate` instead of false, isn't that a change in behaviour?

Reply:
I changed them just in case. Technically they should be `isPrivate` and not `false` so I changed them. However, the change doesn't affect the PR. IE: It doesn't affect the multi-window opening as per the ticket. In fact, I don't see anything affected with or without the change.